google/api_core/operations_v1/abstract_operations_client.py

```diff
@@ -294,13 +293,14 @@ def __init__(
             client_options = client_options_lib.ClientOptions()
 
         # Create SSL credentials for mutual TLS if needed.
-        use_client_cert = bool(
-            util.strtobool(os.getenv("GOOGLE_API_USE_CLIENT_CERTIFICATE", "false"))
-        )
-
+        use_client_cert = os.getenv("GOOGLE_API_USE_CLIENT_CERTIFICATE", "false")
```

Comment:
Might as well us `lower()` in this assignment so you don't have to repeat it below

(It would be great to cast this to a bool like the previous code was doing, but it seems like too much overhead for the very limited use in a very compact LOC range. The string comparison is OK.)

Reply:
Fixed in 614150d26ee14e697f0afab655da794b745bc99e